docs-src/concepts.md

```diff
@@ -424,7 +426,7 @@ status entries corresponding to their name. Both `GatewayStatus` and
 This is a list that includes a type of condition, the status of that condition,
 and the last time this condition changed.
 
-#### Listeners
+##### Listeners
```

Comment:
Why does `Listeners` nest inside `Gateway Status`. I would expect it to be at the same level since it's a spec field.